odl/tomo/backends/skimage_radon.py

```diff
@@ -127,9 +133,12 @@ def skimage_radon_back_projector(sinogram, geometry, vol_space, out=None):
     skimage_range = skimage_proj_space(geometry, vol_space, sinogram.space)
 
     skimage_sinogram = skimage_range.element()
-    skimage_sinogram.sampling(
-        clamped_interpolation(skimage_range, sinogram), bounds_check=False
-    )
+    with writable_array(skimage_sinogram) as sino_arr:
```

Comment:
As above: why remove `bounds_check = False`?

Reply:
Same as above (*I think*, need to check).

Reply:
Yes